src/layouts/partial-cards.html

```diff
@@ -27,7 +27,7 @@
       }
 
       app-header-layout {
-        background-color: #E5E5E5;
+        background-color: var(--paper-card-background-color, #E5E5E5);
```

Comment:
Hmm I took out `--secondary-background-color` yesterday and replaced this one by `--paper-card-background-color`. Shall I put it back? Currently `--secondary-background-color` is not defined in the styles anymore.